js/src/collapse.js

```diff
@@ -50,6 +50,7 @@ const CLASS_NAME_SHOW = 'show'
 const CLASS_NAME_COLLAPSE = 'collapse'
 const CLASS_NAME_COLLAPSING = 'collapsing'
 const CLASS_NAME_COLLAPSED = 'collapsed'
+const CLASS_NAME_WIDTH = 'collapse-horizontal'
```

Comment:
would prefer a var name like `CLASS_NAME_HORIZONTAL` although this works fine too